Wikipedia/Code/DiffHeaderSummaryView.swift

```diff
@@ -19,21 +19,23 @@ class DiffHeaderSummaryView: UIView, Themeable {
     func update(_ viewModel: DiffHeaderEditSummaryViewModel) {
 
         headingLabel.text = viewModel.heading
-        
-        if viewModel.isMinor,
-            let minorImage = UIImage(named: "minor-edit") {
-            let imageAttachment = NSTextAttachment()
-            imageAttachment.image = minorImage
-            let attributedText = NSMutableAttributedString(attachment: imageAttachment)
-            attributedText.addAttributes([NSAttributedString.Key.baselineOffset: -1], range: NSRange(location: 0, length: 1))
-            
-            if let summary = viewModel.summary {
+        headingLabel.accessibilityLabel = viewModel.heading
+
+        if let summary = viewModel.summary {
+            if viewModel.isMinor,
+               let minorImage = UIImage(named: "minor-edit") {
+                let imageAttachment = NSTextAttachment()
+                imageAttachment.image = minorImage
+                let attributedText = NSMutableAttributedString(attachment: imageAttachment)
+                attributedText.addAttributes([NSAttributedString.Key.baselineOffset: -1], range: NSRange(location: 0, length: 1))
+
                 attributedText.append(NSAttributedString(string: "  \(summary)"))
+
+                summaryLabel.attributedText = attributedText
+            } else {
+                summaryLabel.text = viewModel.summary
             }
-            
-            summaryLabel.attributedText = attributedText
-        } else {
-            summaryLabel.text = viewModel.summary
+            summaryLabel.accessibilityLabel = " SUMMARY \(summary)" // TODO LOCALIZE IT
```

Comment:
Reminder about this TODO, although you may not need the additional prefix with the "Edit summary" label above it.

Reply:
done!